cmd/secrets.go

```diff
@@ -47,50 +56,57 @@ func init() {
 		"Delete Secrets that are older than the duration, e.g. [1y2mo3w4d5h6m7s]")
 }
 
-func validateSecretCommandInput(args []string) error {
-
+func validateSecretCommandInput() error {
 	if _, err := parseCutOffDateTime(config.Resource.OlderThan); err != nil {
 		return fmt.Errorf("Could not parse older-than flag: %w", err)
 	}
 	return nil
 }
 
-func executeSecretCleanupCommand(args []string) error {
-	if len(config.Resource.Labels) == 0 {
-		if err := listSecrets(); err != nil {
+func executeSecretCleanupCommand(resources resource.SecretResources, config *cfg.Configuration) error {
+	namespace := resources.GetNamespace()
+	labels := config.Resource.Labels
+	if len(labels) == 0 {
+		secretNames, labels, err := resources.ListSecrets()
+		if err != nil {
 			return err
 		}
+		log.WithFields(log.Fields{
+			"\n - namespace": namespace,
+			"\n - 🔐 secrets": secretNames,
+			"\n - 🎫 labels":  labels,
+		}).Info("Please use labels to select Secrets. The following Secrets and Labels are available:")
 		return nil
 	}
 
-	c := config.Resource
-	namespace := config.Namespace
-
-	log.WithField("namespace", namespace).Debug("Looking for secrets")
+	log.WithField("namespace", namespace).Debug("looking for secrets")
 
-	foundSecrets, err := openshift.ListSecrets(namespace, getListOptions(c.Labels))
+	err := resources.LoadSecrets(getListOptions(labels))
 	if err != nil {
-		return fmt.Errorf("Could not retrieve secrets with labels '%s' for '%s': %w", c.Labels, namespace, err)
+		return fmt.Errorf("could not retrieve secrets with labels '%s' for '%s': %w", labels, namespace, err)
 	}
 
-	unusedSecrets, err := openshift.ListUnusedResources(namespace, foundSecrets)
+	err = resources.FilterUsed()
 	if err != nil {
-		return fmt.Errorf("Could not retrieve unused secrets for '%s': %w", namespace, err)
+		return fmt.Errorf("could not retrieve unused secrets for '%s': %w", namespace, err)
```

Comment:
Why do you start with a lower-case character on these messages (line 86, too), while all the other error messages start with a capital letter?